tasks/section1.yml

```diff
@@ -388,7 +388,7 @@
 # See not_automated.yml
 
 - name: "SCORED | 1.1.8 | AUDIT | Ensure nodev option set on /var/tmp partition"
-  command: /bin/true
+  shell: mount | grep /var/tmp | grep nodev
   register: result
```

Comment:
Audit tasks should register verbose variable name that ends with `_audit`. As you are adding AUDIT checks please change `register: result` to `register: `*`descriptive_check_name_audit`*